src/Couchbase.Lite.Shared/API/Database/Database.cs

```diff
@@ -1148,24 +1113,8 @@ public void Save(Couchbase.Lite.Linq.IDocumentModel model)
         public ListenerToken AddDocumentChangeListener([@NotNull] string id, [@CanBeNull] TaskScheduler scheduler,
```

Comment:
Also needs `ObsoleteAttribute`

Reply:
Same.. Obsolete Attribute is right above.